src/services/jobs/ehr-deletion-job/delete-from-repo-db.js

```diff
@@ -0,0 +1,28 @@
+import { hardDeleteHealthRecordByConversationId } from '../../database/health-record-repository';
+import { hardDeleteAllMessagesByConversationId } from '../../database/message-repository';
+import { setCurrentSpanAttributes } from '../../../config/tracing';
+import { logError, logInfo } from '../../../middleware/logging';
+import { loggerPrefix } from './ehr-deletion-job-common';
+import { S3Service } from '../../storage';
+
+export const permanentlyDeleteEhrFromRepoAndDb = async (healthRecord) => {
+  const { conversationId } = healthRecord;
+
+  setCurrentSpanAttributes({ conversationId });
+
+  try {
+    const s3 = new S3Service(`/${conversationId}/`);
+    await s3.deleteObject();
+
+    await hardDeleteAllMessagesByConversationId(conversationId);
+    await hardDeleteHealthRecordByConversationId(conversationId);
+
+    logInfo(
+      `${loggerPrefix} Successfully deleted health record with Conversation ID ${healthRecord.conversationId} from S3, and associated records within the database.`
+    );
+  } catch (error) {
+    logError(
+      `${loggerPrefix} Failed to delete health record with conversation ID ${healthRecord.conversationId}, more details: - ${error}`
+    );
```

Comment:
```suggestion
    logError(
      `${loggerPrefix} Failed to delete health record with conversation ID ${healthRecord.conversationId}`, error
    );
```

The logError function can take a second argument which is an error object, so I think it might be good to utilise that feature
And if suggest is accepted, the respective expect statement in test can be changed to like this:
```javascript
    expect(logError).toHaveBeenCalledWith(
      `${loggerPrefix} Failed to delete health record with conversation ID ${conversationId}`, new S3ObjectDeletionError(errorMessage)
    );
```

Reply:
Thank you!